src/test/unit/algorithms/CondProbTableTest.cpp

```diff
@@ -180,13 +183,13 @@ namespace nupic {
 
 
   //----------------------------------------------------------------------
-  void CondProbTableTest::testTable(const string& testName, CondProbTable& table, 
-                                    const vector<vector<Real> > & rows)
+  void testTable(const string& testName, CondProbTable& table, 
+                 const vector<vector<Real> > & rows)
   {
   
     // Test the numRows(), numCols() calls
-    TESTEQUAL(numRows(), table.numRows());
-    TESTEQUAL(numCols(), table.numColumns());
+    EXPECT_EQ(numRows(), table.numRows());
```

Comment:
If this fails the test will continue to the next line. Would `ASSERT_EQ` make more sense or do you want to keep going?